test/integration/scheduler_perf/dra.go

```diff
@@ -109,8 +110,12 @@ func (op *createResourceClaimsOp) run(tCtx ktesting.TContext) {
 		workers = 30
 	}
 	workqueue.ParallelizeUntil(tCtx, workers, op.Count, create)
-	if createErr != nil {
-		tCtx.Fatal(createErr.Error())
+	if len(createErrs) > 0 {
+		var errMsgs []string
+		for _, err := range createErrs {
+			errMsgs = append(errMsgs, err.Error())
+		}
+		tCtx.Fatal(strings.Join(errMsgs, "\n"))
```

Comment:
I'm not convinced that this is an improvement.

If there is something wrong with the test that causes many create calls to fail, then it's likely that they are identical. Showing just one error is then better than showing all because it's shorter.

Reply:
Well, I understand what you mean. Because there is only one method called here, multiple goroutines are likely to have the same error.

Reply:
I think the other changes in `CreatePod ` and `CreatePodWithPersistentVolume `  are more reasonable, because when calling multiple methods, these methods may report errors, and all error information should be collected.

Reply:
Rolled back already